examples/server-history.py

```diff
@@ -34,12 +42,17 @@
 
     # enable history for this particular node, must be called after start since it uses subscription
     server.iserver.enable_history(myvar, period=None, count=100)
+
+    # enable history for myobj events generated by myevgen
+    server.iserver.enable_event_history(myobj, myevgen.event, period=None)
```

Comment:
I will check it, even though I don't think it matters for this implementation as a long as all the fields are there.